components/organisms/Breadcrumb.js

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { faAngleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+
+import { useContext } from "react";
+import { LanguageContext } from "../../context/languageProvider";
+import en from "../../locales/en";
+import fr from "../../locales/fr";
+
+/**
+ * breadcrumb component
+ */
+
+const convertBreadcrumb = (string) => {
+  return string
+    .replace(/-/g, " ")
+    .replace(/oe/g, "ö")
+    .replace(/ae/g, "ä")
+    .replace(/ue/g, "ü");
+};
+
+export default function BreadcrumbsReact(props) {
+  const router = useRouter();
+  const [breadcrumbs, setBreadcrumbs] = useState(null);
+
+  useEffect(() => {
+    if (router) {
+      const linkPath = router.asPath.split("/");
+      linkPath.shift();
+      const pathArray = linkPath.map((path, i) => {
+        return {
+          breadcrumb: path,
+          href: "/" + linkPath.slice(0, i + 1).join("/"),
+        };
+      });
+      setBreadcrumbs(pathArray);
+    }
+  }, [router]);
+  if (!breadcrumbs) {
+    return null;
```

Comment:
this is wild, but it looks like it works okay. 
How does it know what the page title is?

Reply:
it uses the routes, and for now we're just pulling the text  from translation based on the route names.  this might need to be tweaked once we have a better idea about how we wanna move forward with the breadcrumb